docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -140,6 +150,30 @@ main().catch(err => {
     process.exit(1);
 });
 ----
+def entry_added(event):
+    print("Entry added with key: %s, value: %s" % (event.key, event.value))
+
+
+def entry_removed(event):
+    print("Entry removed with key:", event.key)
+
+
+def entry_updated(event):
+    print(
+        "Entry updated with key: %s, old value: %s, new value: %s"
+        % (event.key, event.old_value, event.value)
+    )
+
+
+client = hazelcast.HazelcastClient()
+
+map = client.get_map("map").blocking()
+
+predicate = sql("surname = smith")
+
+map.add_entry_listener(predicate=sql("surname = smith"), added_func=entry_added, removed_func=entry_removed, updated_func=entry_updated)
```

Comment:
It would look better if you put each kw argument in its own line:
```
map.add_entry_listener(
    predicate=sql("surname = smith"),
    added_func=entry_added,
    removed_func=entry_removed,
    updated_func=entry_updated
)
```

Reply:
Done.